index.html

```diff
@@ -13,8 +13,8 @@
         "pt":    "pt",
         "ru":    "ru",
         "tr":    "tr",
-        "uk":    "ua",
-        "ua":    "ua",
+        "uk":    "uk",
+        "ua":    "uk",
```

Comment:
Should we remove "ua" from this list?

Reply:
Done. https://github.com/ruby/www.ruby-lang.org/pull/3734